packages/eslint-plugin/src/rules/no-emotion-css/README.md

```diff
@@ -1,11 +1,9 @@
 # @compiled/eslint-plugin/emotion-to-compiled
 
 Ensures usage of the `@compiled/react` library over `@emotion/core` or `@emotion/styled`.
-This rule acts as a form of codemod.
+The `--fix` option [on the command line] automatically fixes problems reported by this rule.
 
-## Examples
-
-👎 Example of **incorrect** code for this rule:
+## Fail
```

Comment:
How come the change of style here? I don't think there is any general recommendation though, but I normally see the thumbs up / down approach

Reply:
It's consistent with the other Compiled rule README now (previously they were different) - it also matches more closely to the official ESLint rule READMEs.